### DISCUSSION TITLE
Converting JSONPATH filtered array into string

### DISCUSSION THREAD

**Discussion by ChrzanowskiDariusz** [stepci/stepci#234] [Q&A]
Hi everyone,

I am capturing result of a filtered query ?(). The result is array with one string element inside it. In my next steps I'm trying to use that captured value as a parameter in url, however it doesn't seem to be able to access this value at is not a string. I was trying to access it both in captures and in next step url as an array index value:

  captures:
    someValue:
      jsonpath: $.data[?(@.alias == "alias")].someValue[0]
or
 url: /something/${{ captures.someValue[0] }}

but none of this was working. Could you provide solution for this?

Thanks

**Comment by tkruber** [comment]
@ChrzanowskiDariusz, did you figure out how to do it? I'm having the exact same issue at the moment!

**Comment by ChrzanowskiDariusz** [comment]
I don't think there is solution for it. We are running those tests as a part of an ado pipeline so we included job before it containing script that does necessary filtering.
